website/content/docs/platform/k8s/helm/run.mdx

```diff
@@ -482,6 +482,28 @@ $ kubectl exec -ti <name of pod> -- vault operator unseal
 After a few moments the Vault cluster should elect a new active primary. The Vault
 cluster is now upgraded!
 
+#### Interaction with Raft Autopilot Automated Upgrades
+
+If `server.ha.raft` is enabled with Vault 1.11+, Autopilot has the ability to do
```

Comment:
nit: can we clarify whether 1.11.0 is the minimum version to upgrade _from_ or _to_? I suspect the former?